.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "developers.home-assistant",
+    "dockerFile": "Dockerfile",
+    "appPort": [
+        3000
+    ],
+    "postCreateCommand": "yarn install",
+    "extensions": [
+        "davidanson.vscode-markdownlint",
+        "eamodio.gitlens",
+        "editorconfig.editorconfig",
+        "github.vscode-pull-request-github",
+        "mrmlnc.vscode-scss",
+        "ms-vsliveshare.vsliveshare",
```

Comment:
This is quite a heavy extension. A user can still install it, but it isn't required for developing on this project?

Reply:
I left it in as I re used the devcontainer from home-assistant/home-assistant.io. I haven't used it myself but thought it was maybe something you were using as a tool

Reply:
Hmm yeah, that should be removed from there as well... Will do that :)